api/dynamic/src/main/kotlin/ReflectionLoader.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2023 Yandex LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.yandex.yatagan.dynamic
+
+import com.yandex.yatagan.AutoBuilder
+import com.yandex.yatagan.base.singleInitWithFallback
+import com.yandex.yatagan.internal.ImplementationLoader
+import com.yandex.yatagan.internal.YataganInternal
+import com.yandex.yatagan.rt.engine.RuntimeEngine
+import com.yandex.yatagan.rt.support.DynamicValidationDelegate
+import com.yandex.yatagan.rt.support.Logger
+
+/**
+ * Instantiated reflectively.
+ * WARNING: Keep class name in sync with [com.yandex.yatagan.Yatagan.DYNAMIC_LOADER_DELEGATE_CLASS_NAME].
```

Comment:
Test for that maybe?

Reply:
It is already covered. Basically, every `[RT]` test with a `fun test()` will fail if this doesn't match.